tests/e2e/concatenate-xpath-strings.spec.js

```diff
@@ -1,42 +0,0 @@
-const helper = require('../helper');
```

Comment:
Yes. I believe that updating our version of `openrosa-xpath-evaluator` broke this functionality since that added an openrosa implementation of `concat`.  

I have made a note of this in the PR doc (including that we should add this to release notes).